spec/grover/processor_spec.rb

```diff
@@ -579,6 +603,46 @@
         it { expect(pdf_text_content).to eq "#{date} http://www.example.net/foo/bar 1/1" }
       end
 
+      context 'when wait for function option is specified with options' do
```

Comment:
Any reason this doesn't live up under the other test you added? Would be good to keep the two together.

Reply:
My logic here was that I put:
 `when wait for function option is specified` right under `when wait for selector option is specified`
 and similarly:
 `... is specified with options` right under `when wait for selector option is specified with options` 
 
 But I do agree that it makes more sense to keep them close together instead